packages/icon-button/src/IconButton.tsx

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
-import { cx } from '@leafygreen-ui/emotion';
 import PropTypes from 'prop-types';
+
+import { validateAriaLabelProps } from '@leafygreen-ui/a11y';
 import Box from '@leafygreen-ui/box';
-import { isComponentType } from '@leafygreen-ui/lib';
+import { cx } from '@leafygreen-ui/emotion';
+import { isComponentGlyph } from '@leafygreen-ui/icon';
 import {
   useDarkMode,
   useUsingKeyboardContext,
 } from '@leafygreen-ui/leafygreen-provider';
-import { isComponentGlyph } from '@leafygreen-ui/icon';
-import { validateAriaLabelProps } from '@leafygreen-ui/a11y';
-import { AccessibleIconButtonProps, IconProps, Size } from './types';
+import { isComponentType } from '@leafygreen-ui/lib';
+
```

Comment:
I don't know how I feel about these line breaks? I think it would be legible enough with the linted groupings.